packages/blade/src/components/Alert/StyledAlert.d.ts

```diff
@@ -1,13 +1,13 @@
 import type { ReactNode } from 'react';
 
-import type { ColorContrast, IntentTypes } from '~tokens/theme/theme';
+import type { ColorContrast, Feedback } from '~tokens/theme/theme';
 
 export type StyledAlertProps = {
   children: ReactNode[];
```

Comment:
Pretty sure you don't need to define ReactNode as array. It already has `ReactNodeArray` defined.

Reply:
Done in https://github.com/razorpay/blade/pull/666/commits/f2ae55867de2cc17e9f039fa0cd99e49df089d25